docs/DeveloperGuide.md

```diff
@@ -4,10 +4,137 @@
+ that can be ordered. Multiple menus can exist and each has a unique ID. <br><br />
+* [**`MenuItem`**](#model-component): A class inheriting item, and represents a food item on the menu. <br><br />
+* [**`Order`**](#model-component): A class representing an order to be entered into the system to be kept track of. Each 
+  order has a unique ID generated from the time of order.<br><br />
+* [**`SetMenu`**](#model-component): An enumeration representing the different types of set menus available, examples of
+  which includes *breakfast*, *lunch*, *dinner*.
+
+The *Class Diagram* below shows how the model components interact with each other, including interactions such as 
+dependencies, associations and inheritance.
+
+![Class Diagram](images/modelcomponent.png)
+
+## Implementation
+
+### `MainLogic`
+Generally, the main logic works as follows:
+1. User enters an input which is received in the *ui* and parsed by the `Parser`.
+2. The `Parser` classifies the command based on `CommandType`.
+3. If it is a first level command, `execute` is called on the corresponding class.
+4. If it is not a first level command, the command will be pass to `SubLogic` to handle.
+
+**Create Order** <br>
+`Mainlogic` takes user input and crate an `Order` class and pass it to `OrderLogic` to execute the command.
+
+**View Order by ID** <br>
+`Mainlogic` takes in the command and the order ID, execute the `view order` command by calling a static method<br>
+in `ViewOrderCommand` class.
+
+**View all orders** <br>
+`Mainlogic` takes in the command and calls the `ViewOrdersSummaryCommand` class to execute the command<br>    
+by querying the orderList.
+
+**View Receipt** <br>
+`Mainlogic` takes in the command and calls the `ViewReceiptCommand` class to execute the command
+### `OrderLogic`
+Generally, the order logic works as follows: 
+1. User enters an input which is received in the *ui* and parsed by the `Parser`. 
+2. The `Parser` classifies the command based on `CommandType`
+3. Within `OrderLogic`, `execute` is called on the corresponding class
+4. Control is passed to other sections of the code
+
+**View Menu** <br>
+Within the construct of the order logic, the menu can be accessed for viewing in order to select items from 
+available menus. This is carried out with the `view menu` command.
+
+**View Item**  
+Within `OrderLogic`, a list containing all the items that have been added to the current active order can be viewed by executing
+the `view item` command.
+
+**Add**  
+Inside `OrderLogic`, items from the menu can be added into the current active order.
+This is carried out using the `add -item <item_id> -quantity <quantity_of_item>` command,
+where `<item_id>` is an integer corresponding to the item's id in the menu,
+and `<quantity_of_item>` is an integer of the amount of that item to be added.
+
+**Delete**  
+In `OrderLogic`, items from the current order can be removed via the
```

Comment:
just a nit: to make it consistent, we may want to standardize using "Inside" or "in"